teach_module.rb

```diff
@@ -0,0 +1,22 @@
+module teach
```

Comment:
[naming of modules](https://rubymonk.com/learning/books/1-ruby-primer/chapters/35-modules/lessons/80-modules-as-namespaces
)